Lab_2.ipynb

```diff
@@ -0,0 +1,691 @@
+      "text/plain": [
+       "<Figure size 1080x576 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "k = np.arange(0, 20, 1)\n",
+    "y = stats.poisson.pmf(k, X, loc = 0)\n",
+    "y = y / np.sum(y)\n",
+    "\n",
+    "day_1 = np.convolve(y, y)\n",
+    "k1 = np.arange(0, len(day_1), 1)\n",
+    "day_i = day_1\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 2, figsize = (15, 8))\n",
+    "\n",
+    "ax[0].step(k1, day_1, label=\"1 convolution\", where='mid')\n",
+    "ax[1].set_yscale(\"log\")\n",
+    "ax[1].step(k1, day_1, label=\"1 convolution\", where='mid')\n",
+    "\n",
+    "convolutions = 4\n",
+    "\n",
+    "for i in range(convolutions):\n",
+    "    day_i = np.convolve(day_i, day_1)\n",
+    "    k_i = np.arange(0, len(day_i), 1) / (i+2) \n",
+    "    ax[0].step(k_i, day_i, label=str(i+2)+\" convolutions\", where='mid')\n",
+    "    ax[1].step(k_i, day_i, label=str(i+2)+\" convolutions\", where='mid')\n",
+    "    \n",
+    "ax[0].legend(loc=0)\n",
+    "ax[0].set_xlim([0, 30])\n",
+    "\n",
+    "ax[1].legend(loc=0)\n",
+    "    \n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ccfa33a-6fcd-497c-94ff-91cf8195801c",
+   "metadata": {},
+   "source": [
+    "The average value stays the same. As you average for more days, the distribution has a finer gradient and begins looking more Gaussian."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07fa1437-b2b8-4791-9332-cf6d3137ad50",
+   "metadata": {},
+   "source": [
+    "##### D)\n",
+    "\n",
+    "Finding the Xσ of detecting 8.9 (Y) gamma rays after 30 days.\n",
+    "\n",
+    "**Statistical question:** what's the chance the cosmic-ray background produced a measurement as signal-like or more than my detected measurement of 8.9 after 30 days? "
```

Comment:
May want to just write what the math would be in words or in an integral to make it explicit.